app/views/caseload/parole_cases.html.erb

```diff
@@ -0,0 +1,11 @@
+<% content_for :title, "Cases approaching parole – Digital Prison Services" %>
+<%= render partial: 'layouts/caseload' %>
+
+<h2 class="govuk-heading-l">Parole cases</h2>
+<p class="govuk-body">Cases with a target hearing date, PED or TED in the next 10 months.</p>
+
+<% if @pom.blank? || @parole_cases.empty? %>
+  <h2 class="govuk-heading-l">No cases approaching parole </h2>
+<% else %>
+  <%= render 'shared/caseload_table', header_partial: 'shared/caseload_headers', collection_partial: 'shared/caseload', collection: @parole_cases %>
```

Comment:
Looking at [the designs](https://hmpps-moic-prototype.apps.live.cloud-platform.service.justice.gov.uk/caseload/overview) I'm not sure it should be using the shared caseload table

It may have changed since, but [here's](https://github.com/ministryofjustice/offender-management-allocation-manager/pull/2132/files#diff-1208bf8f3e4378c466c24f21eb7578dcd9c402a61b9141bb8f0bc3b2903367f5R11-R57) the table I wrote a while back for what I think is this page

If the requirements have changed then of course ignore the above 😁